src/common/components/__tests__/SearchBox.test.jsx

```diff
@@ -22,4 +23,13 @@ describe('SearchBox', () => {
     wrapper.update();
     expect(wrapper).toMatchSnapshot();
   });
+
+  it('calls onSearch on input search', () => {
+    const onSearch = jest.fn();
+    const wrapper = shallow(<SearchBox value="value" onSearch={onSearch} />);
+    const onInputSearch = wrapper.find(Input.Search).prop('onSearch');
+    const searchValue = 'foo';
+    onInputSearch(searchValue);
+    expect(onSearch).toBeCalledWith(searchValue);
```

Comment:
Alternatively `expect(onSearch).toBe(onInputSearch);` can be used but might require modification when we want to extend `onSearch` in the component to do other things.

do you any preference @drjova ?